src/chapters/sql-part2/exercises.rst

```diff
@@ -0,0 +1,97 @@
+#. ``season`` is either ``"Winter"``, ``"Spring"``, ``"Summer"``, or ``"Fall"`` depending on which season our plant blooms in.
+
+In a new query tab, use ``CREATE TABLE`` to make the ``plant`` table.
+After you have created the ``plant`` table, you need to import some data.
+
+Our second table is ``seeds``. ``seeds`` needs to have five columns:
+
+#. ``seed_id``, which will auto increment.
+#. ``expiration_date``, which will be of the ``DATE`` type.
+#. ``quantity`` is an integer for the number of seeds we have available.
+#. ``reorder`` of type ``BOOL`` and tells us if we need to reorder that type of seed.
+#. ``plant_id`` is the foreign key that connects ``seeds`` to ``plant``.
+
+In a new query tab, use ``CREATE TABLE`` to make the ``seeds`` table.
+After you have created the ``seeds`` table, you need to import some data.
+
+Our third table is ``garden_bed``. ``garden_bed`` needs to have four columns:
+
+#. ``space_number``, which will increment as we plant new plants.
+#. ``date_planted`` of type ``DATE`` for the date we planted that plant on.
+#. ``doing_well`` of type ``BOOL`` and tells us if the plant is doing well.
+#. ``plant_id`` is the foreign key that connects ``garden_bed`` to ``plant``.
+
+In a new query tab, use ``CREATE TABLE`` to make the ``garden_bed`` table.
+After you have created the ``garden_bed`` table, you need to import some data.
+
+Try out some joins!
+-------------------
+
+Before we jump in with our subqueries and more complex queries, let's practice our joins from the previous chapter.
+After you write each query, look at the result set and reflect on why that information might be helpful.
+If you needed this database to manage your own garden, what would you use the result set for?
+
+Inner Join
+^^^^^^^^^^
+
+Write a query that joins ``seeds`` and ``garden_bed`` with an inner join to see what plants are we have seeds for that are *also* planted in the garden bed.
+
+Left Join
+^^^^^^^^^
+
+Write a query that joins ``seeds`` and ``garden_bed`` with a left join to see all of the seeds we have and any matching plants in the garden bed.
+
+Right Join
+^^^^^^^^^^
+
+Write a query that joins ``seeds`` and ``garden_bed`` with a right join to see all the plants in the garden bed and any matching seeds we have.
+
+Full Join
+^^^^^^^^^
+
+Write a query that joins ``seeds`` and ``garden_bed`` with a full join.
+
+.. admonition:: Note
+
+   In the previous chapter, we learned that MySQL does not support full joins.
+   Review the section on :ref:`full joins <fullouterjoin>` to see how this could be done!
+
+Sub-Queries and Complex Queries
+-------------------------------
+
+When we were writing our joins, you may have noticed that the information that was most helpful to you (the ``plant_name``) was missing from the result set!
+
+Write a query that gets the name of the plant by joining the ``plant`` table on the result set of the inner join query above.
```

Comment:
What do you think about adding a hint here?

Reply:
Added a hint!